services/api/src/messaging/players.service.ts

```diff
@@ -88,6 +89,28 @@ const moveSkipped: Handler = (ws, { userId }: MoveSkippedMessage): void => {
   }
 };
 
+const removePlayer: Handler = (ws, { userId }: RemovePlayerMessage): void => {
+  const [playerConnection, player] = findPlayerById(userId);
+  try {
+    if (!player) {
+      throw new Error(`Player with id ${userId} not found`);
+    }
+    publish({
+      type: MessageType.RemovePlayer,
+      payload: userId,
+    });
+
+    players.delete(playerConnection);
+    logger.info(`Player ${player?.name} removed`);
+    publish({
+      type: MessageType.NewPlayer,
```

Comment:
Basically it's not NewPlayer, it's just sending all players to the frontend, so it updates the player list, I wasn't sure if we need to create a separate method UpdatePlayerList if it works in the same way like NewPlayer method

Reply:
Sending `NewPlayer`event to refresh player list sounds like a workaround. I'd prefer a separate action, so that we would not need to worry about `NewPlayer`event firing unexpectedly.

Reply:
Changed NewPlayer event into UpdatePlayerList, because NewPlayer doesn't send just one new player, it sends all players, so it's better to call it "UpdatePlayerList" in my opinion